crypto/conf/conf_mod.c

```diff
@@ -481,7 +481,7 @@ char *CONF_get1_default_config_file(void)
     int len;
 
     if (!OPENSSL_issetugid()) {
-        file = getenv("OPENSSL_CONF");
+        file = ossl_safe_getenv("OPENSSL_CONF");
```

Comment:
See the dissonance? One line up? Now jumping to conf_api. If config file. i.e. one that is *not* appointed by OPENSSL_CONF, is considered trusted, would it be appropriate to deny environment references?